cms/djangoapps/contentstore/views/organization.py

```diff
@@ -18,6 +20,12 @@ class OrganizationListView(View):
     @method_decorator(login_required)
     def get(self, request, *args, **kwargs):
         """Returns organization list as json."""
-        organizations = get_organizations()
-        org_names_list = [(org["short_name"]) for org in organizations]
+
+        if not waffle.switch_is_active(settings.ENABLE_EDLY_ORGANIZATIONS):
```

Comment:
move this check to `get_edly_organizations` also change the name to a more generic one like `get_enabled_organizations`
here we will have only one line change:
```python
organizations = get_enabled_organizations()
```